src/components/Header.vue

```diff
@@ -1,40 +1,68 @@
 <script lang="ts">
     import { defineComponent } from 'vue';
+    import CartFlyout from './CartFlyout.vue';
+    import { CartItemInterface } from '../types';
 
     export default defineComponent({
-    name: 'Header',
-    props: {
-        mobileMenuOpen: Boolean,
-        cartOpen: Boolean,
-        totalQuantity: {
-            type: Number,
-            default: 0,
-        },
-    },
+      components: {CartFlyout},
+      name: 'Header',
+      props: {
+          mobileMenuOpen: Boolean,
+          cartOpen: Boolean,
+          totalQuantity: {
+              type: Number,
+              default: 0,
+          },
+          cartItems: {
+            type: Array as () => CartItemInterface[],
+            required: true,
+          },
+      },
+      setup(props, context) {
+        const handleRemoveFromCart = (itemId: number) => {
+          console.log('remove from cart', itemId);
+          context.emit('remove-from-cart', itemId);
+        };
+
+        return {
+          handleRemoveFromCart,
+        };
+      },
     });
 </script>
 
 <template>
-    <header class="py-5 px-6 flex justify-between items-center">
-      <div class="flex justify-between items-center gap-4">
-        <div class="md:hidden">
-          <button class="flex justify-center items-center" @click="$emit('toggle-mobile-menu')">
-            <img src="../assets/icon-menu.svg" alt="menu" />
-          </button>
-        </div>
-        <img class="self-center pb-1" src="../assets/logo.svg" alt="sneakers" />
+    <header class="py-5 md:py-0 px-6 md:px-0 flex justify-between items-center border-b-neutral-grey border-b-2 relative">
```

Comment:
On tablet, your header doesn't have paddings from sides, it shouldn't stick to window's frame
![Screenshot from 2023-04-24 10-17-32](https://user-images.githubusercontent.com/36459922/233925578-3cb796b1-18bb-4388-b878-6656d63ba705.png)